CourseGrab/ViewControllers/Search/SearchDetailTableViewCell.swift

```diff
@@ -49,14 +71,16 @@ class SearchDetailTableViewCell: UITableViewCell {
 
     func configure(title: String, isOpen: Bool, isTracked: Bool) {
```

Comment:
While you plan on refactoring this in "next steps", perhaps already changing the `configure` method to take in a `Session` model and then set the corresponding UI with that would be better. Feel free to add an `isTracking`bool to the struct for the time being.